py3-nbclassic/shlex.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/setupbase.py b/setupbase.py
```

Comment:
Could we add a comment above the diff explaining why we vendor this patch?

Reply:
Thanks RJ, addressed it via https://github.com/wolfi-dev/os/pull/37244/commits/f3f570617a5c9ff4916a34aac7a1e619c703eb4d